applications/vessel-history/src/features/settings/components/Encounters.tsx

```diff
@@ -0,0 +1,168 @@
+
+  const eezs = EEZ_OPTIONS.filter((option) => settings.eezs?.includes(option.id))
+  const rfmos = RFMOS_OPTIONS.filter((option) => settings.rfmos?.includes(option.id))
+  const mpas = EEZ_OPTIONS.filter((option) => settings.mpas?.includes(option.id))
+
+  return (
+    <div className={styles.settingsFieldsContainer}>
+      <div className={styles.settingsField}>
+        <label>
+          Events in There EEZs
+          <IconButton type="default" size="tiny" icon="info" tooltip="testing tootip"></IconButton>
+        </label>
+        <MultiSelect
+          selectedOptions={eezs}
+          placeholderDisplayAll={true}
+          onCleanClick={() => setSettingOptions(section, 'eezs', [])}
+          onSelect={(selected) => setSettingOptions(section, 'eezs', [...eezs, selected])}
+          onRemove={(option) =>
+            setSettingOptions(
+              section,
+              'eezs',
+              eezs.filter((o) => o.id !== option.id)
+            )
+          }
+          options={EEZ_OPTIONS}
+        ></MultiSelect>
+      </div>
+      <div className={styles.settingsField}>
+        <label>
+          Events in these RFMOS
+          <IconButton type="default" size="tiny" icon="info" tooltip="testing tootip"></IconButton>
+        </label>
+        <MultiSelect
+          onCleanClick={() => setSettingOptions(section, 'rfmos', [])}
+          selectedOptions={rfmos}
+          placeholderDisplayAll={true}
+          onSelect={(selected: MultiSelectOption) =>
+            setSettingOptions(section, 'rfmos', [...rfmos, selected])
+          }
+          onRemove={(option) =>
+            setSettingOptions(
+              section,
+              'rfmos',
+              rfmos.filter((o) => o.id !== option.id)
+            )
+          }
+          options={RFMOS_OPTIONS}
+        ></MultiSelect>
+      </div>
+      <div className={styles.settingsField}>
+        <label>
+          Events in these MPAs
+          <IconButton type="default" size="tiny" icon="info" tooltip="testing tootip"></IconButton>
+        </label>
+        <MultiSelect
+          onCleanClick={() => setSettingOptions(section, 'mpas', [])}
+          selectedOptions={mpas}
+          placeholderDisplayAll={true}
+          onSelect={(selected: MultiSelectOption) =>
+            setSettingOptions(section, 'mpas', [...mpas, selected])
+          }
+          onRemove={(option) =>
+            setSettingOptions(
+              section,
+              'mpas',
+              mpas.filter((o) => o.id !== option.id)
+            )
+          }
+          options={EEZ_OPTIONS}
+        ></MultiSelect>
+      </div>
+      <div className={cx(styles.settingsField, styles.inlineRow)}>
+        <label>
+          DURATION
+          <IconButton type="default" size="tiny" icon="info" tooltip="testing tootip"></IconButton>
+        </label>
+        <span>Longer than</span>
+        <InputText
+          type="number"
+          value={settings.duration ?? 2}
+          min={2}
```

Comment:
Why 2 is the min? As far as I could read in the ticket it should be 1
> Range to select goes from 1 hour to 99 hours.
Use constants for these ranges too.